defaults/main.yml

```diff
@@ -31,11 +31,40 @@ omero_web_config_set: {}
 # Install and configure Nginx
 omero_web_setup_nginx: true
 
+# Python 3?
+omero_web_python3: false
+
+# List of additional Python packages to be installed into virtualenv
+# (python 3 only)
+omero_web_python_addons: []
+
 
 ######################################################################
 # Expert users only!
 ######################################################################
 
+# List of python3 packages to install
+omero_web_python_requirements:
+  - https://github.com/ome/zeroc-ice-py-centos7/releases/download/0.2.1/zeroc_ice-3.6.5-cp36-cp36m-linux_x86_64.whl
+  - omero-py==5.6.dev2
+  - omero-web==5.6.dev3
+  - redis
+
+# Path to virtualenv
+omero_web_virtualenv_basedir: >-
+  {{
+    omero_web_python3 | ternary(
+      omero_web_basedir + '/venv3',
```

Comment:
I understand a benefit of using a new directory is to have a cleaner migration, but there's a good possibility that end-user scripts will need updating.